File: my-fresh-copy/hw/palomo/u2/hw09StoreApp/JavaScript/StoreApp/ec.edu.espe.product.model/product.js
/* 
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/ClientSide/javascript.js to edit this template
 */
import { Tax } from "../../taxes-lib/tax.js";

export class Product {
    constructor(id, description, price, pvp = null) {
        this.id = id;
        this.description = description;
        this.price = price;
        if (pvp === null) {
            this.pvp = Tax.computeTotal(price, 15.0);
        } else {
            this.pvp = pvp;
        }
    }

    toString() {
        return `Product\n{id=${this.id}, description='${this.description}', price=${this.price}, pvp=${this.pvp}}`;
    }
}

File: ws/basurto/u3/ws26Interface/AnimalFarmSystem/src/main/java/ec/edu/espe/animalfarmsystem/model/Product.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.animalfarmsystem.model;

/**
 *
 * @author Esteban Basurto, CodeBreakers, @ESPE
 */
public class Product {
    private int id;
    private String description;
    private String quantify;
    private String unit;

    @Override
    public String toString() {
        return "Product{" + "id=" + id + ", description=" + description + ", quantify=" + quantify + ", unit=" + unit + '}';
    }

    public Product(int id, String description, String quantify, String unit) {
        this.id = id;
        this.description = description;
        this.quantify = quantify;
        this.unit = unit;
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * @return the quantify
     */
    public String getQuantify() {
        return quantify;
    }

    /**
     * @param quantify the quantify to set
     */
    public void setQuantify(String quantify) {
        this.quantify = quantify;
    }

    /**
     * @return the unit
     */
    public String getUnit() {
        return unit;
    }

    /**
     * @param unit the unit to set
     */
    public void setUnit(String unit) {
        this.unit = unit;
    }
    
}